packages/terra-demographics-banner/README.md

```diff
@@ -0,0 +1,25 @@
+# Terra Demographics Banner
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-demographics-banner.svg)](https://www.npmjs.org/package/terra-demographics-banner)
+[![Build Status](https://travis-ci.org/cerner/terra-core.svg?branch=master)](https://travis-ci.org/cerner/terra-core)
+
+The demographics component is used to display demographic information about a person in a condensed, easy to read format.
+
+- [Getting Started](#getting-started)
+- [Documentation](https://github.com/cerner/terra-ui/tree/master/packages/terra-demographics-banner/docs)
```

Comment:
Rats. Thought I hit them all correctly in my previous commit. Any rate, it's fixed here. https://github.com/cerner/terra-core/pull/280/commits/3f362fbb4b6253e68b965940ee54a7576d0a1c9f